packages/middleware/src/helpers/getAgnosticStatusCode.ts

```diff
@@ -68,6 +68,11 @@ function getApolloStatusCode(error: ApolloError) {
   if (error.code) {
     return typeof error.code === "string" ? 400 : error.code;
   }
+
+  if (error?.graphQLErrors?.length > 0) {
+    return 200;
```

Comment:
The only question I have, do we want to return 200 in case of graphql errors?

Reply:
yes. this is not a response/request error